### PR TITLE
Added additional information to search report page

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added additional information to search report page

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/search-terms-report.html

### CODE REVIEW

File: src/marketing/search-terms-report.md [hunk 1]
```diff
@@ -12,3 +12,24 @@ The Search Terms report shows the number of results for each term, and the numbe
...
+
+### Grid Column descriptions
+
+|Column|Description|
+|--- |--- |
+|ID|Unique, numeric ID generated for the search term entry|
+|Search Query|A query the search was performed with|
+|Store|The store were the search query was performed|
+|Results|Number of results|
+|Hits|Number of Uses|
```
Comment (leslietilling) [inline src/marketing/search-terms-report.md:35]:
```suggestion
## Export the Search Terms report

1. For **Export to**, choose an export format:

    - **CSV** - A comma-separated value file containing plain text data
    - **Excel XML** - An XML-based, spreadsheet data format

1. Click <span class="btn">Export</span>.

The generated file is automatically saved to your designated folder for downloads.

## Column descriptions

|Column|Description|
|--- |--- |
|ID|Unique, numeric ID generated for the search term entry|
|Search Query|The query used to perform the search|
|Store|The store where the search query was applied|
|Results|Number of results|
|Hits|Number of uses|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for style and clarity

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**